ViewModel/Study/FDACache.cs

```diff
@@ -815,6 +858,10 @@ public List<ChildElement> GetChildElementsOfType(Type childElementType)
                 }
                 return retVal;
             }
+            else if(childElementType == typeof(AlternativeComparisonReportElement))
```

Comment:
Another duplicate above, and they don't use else if or AddRange

Reply:
removed the duplicate.